db/migrations/20240529151056_Page.sql

```diff
@@ -0,0 +1,9 @@
+-- Ensure the UUID extension is enabled
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE Page (
+    page_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    user_id Uuid UNIQUE,
+    url_id Uuid UNIQUE,
```

Comment:
Do we need this `url_id` field?

I typically only add `x_id` columns when we have a table for whatever it is. So since `url` is 'just' a column on `Page` and not a table itself I don't think we'll need this `url_id`!